### ISSUE TITLE
Regression: "Q" key does not close the pager any more

### ISSUE THREAD

**Issue by sergiimk** [arijit79/minus#56]
**Describe the bug**
Pressing "Q" does not exit the pager any more.

I've bisected the versions and the regression occurs between versions `4.0.1` and `4.0.2`.

**Expected behavior**
"Q" key exits as documented

**Desktop:**
 - OS: `Arch Linux`
- Terminal: `gnome-terminal`
- Shell: `zsh`
- `minus` Version: `4.0.3` (latest)

Thank you for this library!

**Comment by arijit79** [comment]
Thanks for filling up the issue. I will definitely look into it.

**Comment by arijit79** [comment]
In my tests, `q` works as expected. Can you give some more info on the configuration. Also uppercase doesn't work, if that's the case that could be added to the `handle_input()` function.

**Comment by sergiimk** [comment]
Hi @arijit79, sorry for delayed response - I finally had time to take a closer look.

Turns out the problem only reproduces itself under `ExitStrategy::PagerQuit`. Here's the minimal example I used:

```rust
use std::fmt::Write;

fn main() {
    let mut pager = minus::Pager::new().unwrap();
    pager.set_exit_strategy(minus::ExitStrategy::PagerQuit);
    pager.set_prompt("Test");

    pager.write_str("Hello!\nWorld!\n").unwrap();

    minus::page_all(pager).unwrap();
}
```

I've used `git bisect` to trace the breakage to this commit: f3ab74cf955bca00b1d0b0ed3ade1de0fd4390de

Debugging the code I see that the `static_paging()` loop correctly invokes `term::cleanup()` function when I hit `q` key, but if we're not terminating the process on exit - there simply no condition that tells `static_paging()` loop should exit, so it keeps running forever.
